scripts/schema/config/solver.json

```diff
@@ -7,6 +7,8 @@
   "properties":
   {
     "Order": { "type": "integer", "minimum": 1 },
+    "PartialAssemblyOrder": { "type": "integer", "minimum": 1 },
+    "Device": { "type": "string" },
```

Comment:
Yeah for now this is string-valued since the list can get quite long (see https://mfem.org/howto/assembly_levels/). Perhaps before merging, or perhaps in a follow on, adding better default values and also allowing for some restricted enum interface for end users is the way to go. I'm not sure if we want to add this here or instead merge PA as turned off by default, with the string `device` value, and later on (GPU?) activate it by default with some better UI.